src/storage/v2/schemas.hpp

```diff
@@ -31,10 +32,8 @@ class SchemaViolationException : public utils::BasicException {
   using utils::BasicException::BasicException;
 };
 
-struct SchemaProperty {
-  enum class Type : uint8_t { Bool, Int, Double, String, Date, LocalTime, LocalDateTime, Duration };
-
-  Type type;
+struct SchemaPropertyType {
+  common::SchemaType type;
```

Comment:
SInce `SchemaProperty` has been removed from Schema definition, this was renamed to `SchemaType`, we had this discussion @antaljanosbenjamin

Reply:
I would really like to avoid the name `Property` since it is heavily used it always represents a value, not a type. How about `DataType`? (inspired by postgres docs https://www.postgresql.org/docs/current/sql-createtable.html)

Reply:
Sorry I misunderstood, I agree SchemaProperty is a better name for this struct